### PR TITLE
Reword button guidelines here and there

### PR DESCRIPTION
**Pull request by huayunh**
Final touch of BLUI-3592

#### Changes proposed in this Pull Request:
- Wordsmith

#### To Test:
- goto http://localhost:3000/patterns/buttons

### CODE REVIEW

**Comment by huayunh** [comment]
I need this to be approved before Monday so that I can use it for user testing. Thanks.

File: src/docs/patterns/buttons.mdx [hunk 1]
```diff
@@ -219,15 +229,22 @@ Buttons are often used within other components.
...
     ]}
 />
 
 ## Placement
 
-People generally read from left to right and from top to bottom. You should place buttons in the order you want them to be read.
+You should place buttons in the order you want them to be read. Most languages are read from left to right and from top to bottom.
+
+<MaterialDesignDescription
+    title={'Internationalization § Bidirectionality / Right-to-Left (RTL) Support'}
```
Comment (emclaug2) [inline src/docs/patterns/buttons.mdx:243]: Huh, this is my first time seeing this symbol `§` - was this intentional?

**Review by emclaug2 — APPROVED** [review]

File: src/docs/patterns/buttons.mdx [hunk 2]
```diff
@@ -123,16 +125,16 @@ You should arrange your buttons so that users’ attention can be guided to enco
...
+        `DO: Use buttons styles to appropriately guide users' attention.`,
         `DONT: Do not place a “text-only” button right next to a "filled" or an "elevated" button, as the text-only button can be neglected by users in a hurry.`,
         `DO: Use outlined buttons next to filled buttons.`,
     ]}
 />
 
 ## Icons
 
-Buttons are often combined with icons to help manage users’ attention, especially if the user is not a native speaker.
+Buttons are often combined with icons to draw users’ attention. Icons also help reinforce the button actions, especially in multilingual applicatons.
```
Comment (jeffvg) [inline src/docs/patterns/buttons.mdx:137]: just typo here...applications

Comment (daileytj) [inline src/docs/patterns/buttons.mdx:137]:
```suggestion
Buttons are often combined with icons to draw users’ attention. Icons also help reinforce the button actions, especially in multilingual applications.
```

**Review by daileytj — APPROVED** [review]

**Review by jeffvg — APPROVED** [review]

File: src/docs/patterns/buttons.mdx (hunk 2 above)
Comment (huayunh) [reply src/docs/patterns/buttons.mdx]: awww thanks!

**Pull request merged**